pandas/core/indexes/multi.py

```diff
@@ -691,11 +691,15 @@ def _format_native_types(self, na_rep='nan', **kwargs):
             new_levels.append(level)
             new_labels.append(label)
 
-        # reconstruct the multi-index
-        mi = MultiIndex(levels=new_levels, labels=new_labels, names=self.names,
-                        sortorder=self.sortorder, verify_integrity=False)
-
-        return mi.values
+        if len(new_levels) == 1:
```

Comment:
just turn this into an Index (and call _format_native_types) on that

Reply:
Gotcha.  I took this as a desire to make this clear we'll be treating this a normal index since the outputs should be the same.  a23252385e9de1a2af0832a82e168d825315d937